zipline/data/loader.py

```diff
@@ -182,19 +182,13 @@ def load_market_data(bm_symbol='^GSPC'):
     # we have data,then we need to update
     if len(days_up_to_now) - last_bm_date_offset > 1:
         benchmark_returns = update_benchmarks(bm_symbol, last_bm_date)
-        if (
-            benchmark_returns.index.tz is None
-            or
-            benchmark_returns.index.tz.zone != 'UTC'
-        ):
+        if benchmark_returns.index.tz is None or \
```

Comment:
I agree that breaking a line on a binop should leave the op on the first line; unfortunately, the pep8 tool forgets this line of pep8

> But most importantly: know when to be inconsistent -- sometimes the style guide just doesn't apply. When in doubt, use your best judgment. Look at other examples and decide what looks best. And don't hesitate to ask!